core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -156,8 +156,9 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
   val platform: ScallopOption[String] =
     opt[String](required = false,
       descr = "Cluster platform where Spark CPU workloads were executed. Options include " +
-        s"${PlatformNames.getAllNames.mkString(", ")}. Default is ${PlatformNames.ONPREM}.",
-      default = Some(PlatformNames.ONPREM))
+        s"${PlatformNames.getAllNames.mkString(", ")}. " +
+        s"Default is ${PlatformFactory.getDefault.getName}.",
+      default = Some(PlatformFactory.getDefault.getName))
```

Comment:
nit: same as before. Do we need to create new instances for `getName`?

Reply:
Updated the flow to use `PlatformNames.DEFAULT`.